### PR TITLE
[EPMRPP-49051] Getting parameters from examples

### PR DESCRIPTION
**Pull request by ihar-kahadouski**

### CODE REVIEW

File: src/main/java/com/epam/reportportal/cucumber/Utils.java [hunk 1]
```diff
@@ -253,13 +251,29 @@ public static TestCaseIdEntry getTestCaseId(Match match, String codeRef) {
...
 	}
 
+	static List<ParameterResource> getParameters(Match match) {
+		List<ParameterResource> parameters = Lists.newArrayList();
+		Optional<String> parameterType = Optional.empty();
+
+		String text = match.getLocation();
+		Matcher matcher = Pattern.compile(PARAMETER_TYPE_REGEX).matcher(text);
+		if (matcher.find()) {
+			parameterType = Optional.of(text.substring(matcher.start() + 1, matcher.end() - 1));
```
Comment (HardNorth) [inline src/main/java/com/epam/reportportal/cucumber/Utils.java]: What about 2 parameters with different types, e.g. `Given I have <int> <string> in my pocket` (6, apples)?

File: src/test/java/com/epam/reportportal/cucumber/UtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+/**
+ * @author <a href="mailto:ihar_kahadouski@epam.com">Ihar Kahadouski</a>
+ */
+public class UtilsTest {
```
Comment (HardNorth) [inline src/test/java/com/epam/reportportal/cucumber/UtilsTest.java]: You verified that your extraction of parameters is correct. But what about sending them to the client? If I delete this line the functionality of parameters will be broken, but your tests will still pass:
https://github.com/reportportal/agent-java-cucumber/blob/7581c29d8159597fa1949cfcc1d81251d4390e45/src/main/java/com/epam/reportportal/cucumber/StepReporter.java#L70

Please add an integration unit-tests.

File: src/main/java/com/epam/reportportal/cucumber/Utils.java (hunk 1 above)
Comment (ihar-kahadouski) [reply src/main/java/com/epam/reportportal/cucumber/Utils.java]: done

File: src/test/java/com/epam/reportportal/cucumber/UtilsTest.java (hunk 2 above)
Comment (ihar-kahadouski) [reply src/test/java/com/epam/reportportal/cucumber/UtilsTest.java]: added

**Pull request merged**